src/Resources/app/storefront/snippet/SnippetFile_de_DE.php

```diff
@@ -4,6 +4,9 @@
 
 use Shopware\Core\System\Snippet\Files\SnippetFileInterface;
 
+/**
+ * @SuppressWarnings(PHPMD.CamelCaseClassName)
```

Comment:
We don't have to stick to the convention of the class name given in Shopware's documantation.
When we implement `SnippetFileInterface` it is enough.
When you change the class name to ex. `SnippetFileDe` you can remove a `@SuppressWarnings` then.